src/modules/storefront/components/Products.tsx

```diff
@@ -0,0 +1,202 @@
+        {clampedProducts?.map((p) => (
+          <Card key={p.id} product={p} card={card} button={button} />
+        ))}
+      </div>
+    );
+  };
+
+  const SliderProducts = () => {
+    return (
+      <Swiper
+        navigation
+        modules={[Navigation, Autoplay]}
+        autoplay={autoSlide ? { delay: 2500 } : false}
+        breakpoints={{
+          640: { slidesPerView: 1, spaceBetween: 10 },
+          768: { slidesPerView: 2, spaceBetween: 10 },
+          1024: { slidesPerView: 3, spaceBetween: 10 },
+          1280: { slidesPerView: 4, spaceBetween: 10 },
+        }}
+        className="pw-max-w-[1500px] md:pw-px-6"
+      >
+        {clampedProducts?.map((p) => (
+          <SwiperSlide key={p.id} className="pw-flex pw-justify-center">
+            <Card key={p.id} product={p} card={card} button={button} />
+          </SwiperSlide>
+        ))}
+      </Swiper>
+    );
+  };
+
+  return (
+    <div className="pw-font-poppins pw-p-10">
+      <h2>{data.title}</h2>
+
+      <div className="pw-flex pw-justify-center">
+        {layout === 'grid' ? <GridProducts /> : <SliderProducts />}
+      </div>
+    </div>
+  );
+};
+
+const Card = ({
+  product,
+  card,
+  button,
+}: {
+  product: Product;
+  card: ProductsDefault['card'];
+  button: ProductsDefault['button'] & { text: string };
+}) => {
+  const imgSize = 260;
+  // const router = useRouter();
+
+  return (
+    <div
+      style={
+        {
+          border: '1px solid #DCDCDC',
+          boxShadow: '2px 2px 10px rgba(0, 0, 0, 0.08)',
+          '--products-card-hover-color': card.hoverColor,
+        } as CSSProperties
+      }
+      className={`pw-w-[296px] pw-p-[18px] pw-rounded-[20px] pw-bg-white product-card`}
+      onClick={() => {
+        // router.push(card.url);
+      }}
+    >
+      <div className="pw-flex pw-justify-center">
+        <img src={product.img} width={imgSize} height={imgSize} />
+      </div>
+      {card.name && <p>{product.name}</p>}
+      {card.description && (
+        <p className="pw-text-[#7E7E7E]">{product.description}</p>
+      )}
+      {card.category && (
+        <p className="pw-text-[#C63535] pw-font-semibold">{product.category}</p>
+      )}
+      {card.price && (
+        <p className="pw-font-bold pw-text-lg">
+          <span className="pw-text-sm pw-pr-2">R$</span>
+          {product.price}
+        </p>
+      )}
+      {card.button && (
+        <button
+          style={
+            {
+              boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.26)',
+              color: button.textColor,
+              '--products-button-hover-color': button.hoverColor,
+              '--products-button-bg-color': button.bgColor,
+            } as CSSProperties
+          }
+          className={`pw-w-full pw-border-b pw-border-white pw-py-2 pw-font-medium pw-rounded-[48px] product-card-button`}
+        >
+          {button?.text}
+        </button>
+      )}
+    </div>
+  );
+};
+
+const fetchProductsByTagAndOrder = async (
```

Comment:
Se isso no futuro for a lógica de chamada na api eu faria tambem em outro arquivo, esse ja esta totalmente acoplado com outros arquivos, seguindo padrao que o SDK utiliza, colocando a chamada dentro de um hook e tudo mais onde retorna a api... no caso nao precisa por enquanto, mas ja é possivel, até colocar dentro de um useQuery para ai depois só trocar esse código para a chamada da api